lib/aws_codegen/rest_service.ex

```diff
@@ -266,7 +269,8 @@ defmodule AWS.CodeGen.RestService do
           "GET" ->
             case language do
               :elixir ->
-                2 + length(request_header_parameters) + length(query_parameters)
+                2 + length(request_header_parameters) + length(request_headers_parameters) +
+                  length(query_parameters)
```

Comment:
Note that I don't believe this arity calculation is correct. However, it is unused for elixir, and I am restoring it to the state that was originally merged in #76. I don't believe the calculation is correct even _before_ that change, but I am not 100% certain.

I don't think the erlang arity calculation needs to change.

Reply:
This has been here for many years by now judging by the `git blame` and was first introduced in https://github.com/aws-beam/aws-codegen/commit/c8d14684a082b3886a91f6278e17af016508aea2 and I really don't feel like touching that now. At least not as part of this PR so good that you left this behind 👍